jni_gen_test/tool/gen.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:io';
+import 'package:jni_gen/jni_gen.dart';
+
+void main(List<String> arguments) async {
+  final javaPath = 'java/';
+  await Process.run('javac', ['dev/dart/jni_gen_test/Example.java'],
+      workingDirectory: javaPath);
+  try {
+    await runTask(JniGenTask(
+        summarySource: SummarizerCommand(
+          sourcePaths: [Uri.directory(javaPath)],
+          classPaths: [Uri.directory(javaPath)],
+          classes: ['dev.dart.jni_gen_test.Example'],
+        ),
+        outputWriter: FilesWriter(
+            cWrapperDir: Uri.directory('src/'),
+            dartWrappersRoot: Uri.directory('lib/'),
+            libraryName: 'jni_gen_test')));
```

Comment:
nit: Add comma's between the closing brackets so that the formatting becomes all 2 spaces indented.

It makes the code easier to read.